modules/swagger-codegen/src/main/resources/typescript-angular2/api.mustache

```diff
@@ -205,9 +209,13 @@ export class {{classname}} {
             body: formParams.toString(),
 {{/hasFormParams}}
             search: queryParameters,
-            withCredentials:true
+            withCredentials:isWithCredentials
```

Comment:
Yes.
Javascript has confusing ideas about "truthy" and "falsy". https://developer.mozilla.org/en-US/docs/Glossary/Falsy